source/operations/cert-manager.rst

```diff
@@ -0,0 +1,577 @@
+.. _minio-certmanager:
+
+============
+cert-manager
+============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+TLS certificate management with cert-manager
```

Comment:
it looks weird as heck but should we `cert-manager` the header?

Reply:
Their own docs use "cert-manager" in body text. Lowercase even at the beginning of a sentence.